app/Http/Controllers/SeatController.php

```diff
@@ -0,0 +1,168 @@
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        return $this->seatRepository->registerSeats($request->all());
+    }
+
+    public function collectUsers($users)
+    {
+        $result = collect();
+        foreach ($users as $user) {
+            $mergeUser = new \stdClass();
+            $mergeUser->id = $user->id;
+            $mergeUser->name = $user->name;
+            $mergeUser->avatar =  $user->avatar ?
+                                    Storage::url(config('site.user.image') . $user->avatar) :
+                                    config('site.default-image');
+
+            $schedule = new \stdClass();
+            $schedule->date = Carbon::parse($user->date)->format('d-m-Y');
+            $schedule->shift = $user->shift;
+
+            $seat = new \stdClass();
+            if (!isset($result[$user->seat_id])) {
+                $seat->id = $user->seat_id;
+                $seat->name = $user->seat_name;
+                $mergeUser->schedules = collect();
+                $mergeUser->schedules->put($schedule->date, $schedule);
+                $temp = collect();
+                $temp->put('' . $user->id . '', $mergeUser);
+                $seat->users = $temp;
+                $result->put('' . $user->seat_id .'', $seat);
+            } else {
+                if (isset($result[$user->seat_id]->users[$user->id])) {
+                    if (isset($result[$user->seat_id]->users[$user->id]->schedules[$schedule->date])) {
+                        $schedule->shift = config('site.shift.all');
+                        $result[$user->seat_id]->users[$user->id]->schedules[$schedule->date] = $schedule;
+                    } else {
+                        $result[$user->seat_id]
+                        ->users[$user->id]
+                        ->schedules->put(
+                            $schedule->date,
+                            $schedule
+                        );
+                    }
+                } else {
+                    $mergeUser->schedules = collect();
+                    $mergeUser->schedules->put($schedule->date, $schedule);
+                    $result[$user->seat_id]->users->put($user->id, $mergeUser);
+                }
+            }
+        }
+
+        return $result;
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/SeatController.php</br>**Line**: 111</br>String "-" does not require double quotes; use single quotes instead